cns/restserver/ipam.go

```diff
@@ -25,7 +25,10 @@
 	)
 
 	err = service.Listener.Decode(w, r, &ipconfigRequest)
-	logger.Request(service.Name, &ipconfigRequest, err)
+	operationName := "requestIPConfigHandler"
+	desiredIPAddress := ipconfigRequest.DesiredIPAddress
+	orchestratorContext := string(ipconfigRequest.OrchestratorContext)
+	logger.Request(service.Name+operationName, desiredIPAddress+orchestratorContext, err)
```

Comment:
Instead can you expose a String function for the struct IPConfigRequest? Concatenating the strings this way will be hard to interpret data